### PR TITLE
build: Add `Parcel`

### LINKED ISSUE #1: POK-1- 🏗 Scaffold - build: Add `Parcel`
# Add `Parcel` Build Tooling

## Acceptance Criteria
- Adds `Parcel` build tool
- Adds `index.js` file
- Adds `index.html` file
- Adds `styles.css` file
- Demonstrates that all file types are connected and working together
- Updates `README.md` with section about running application locally with appropriate NPM scripts

## Installing `Parcel` globally
```bash
npm install -g parcel
```

### PR DESCRIPTION
**Pull request by mathiuskitchens**
Added base index, js, and css files.
Added Parcel build.
Connected all files and confirmed working.
Unsure on what to add to README...

[optional footer(s)]

# Description

See above

## Type of change

- [ ] - fix: - patches a bug in the codebase
- [ ] - feat: - introduces a new feature
- [x] - build: - relates to build tooling / project scaffolding
- [ ] - chore: - non-code related changes, like adding packages / libraries
- [ ] - docs: - changes related to documentation

## Related Issue

Does this change have a related issue in the project?

- [x] - Yes - Closes #1 
- [ ] - No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,7 @@ _The Conventional Commit template before adding content, viewed in Vim editor_
 <img width="703" alt="Screen Shot 2023-01-28 at 12 53 16 PM" src="https://user-images.githubusercontent.com/12022922/215283102-a989033d-1ef6-4c88-be1d-24eed49ba618.png">
 
 _The Conventional Commit template after being filled out with all mandatory information_
+
+## Running Locally
+
+No idea what to write here, please help?
```
Comment (wijohnst) [inline README.md]: Just need to add the command for running the application locally. And it should be formatted as a code snippet, so prefixed and postfixed with three back ticks (```). 

```
npm run start
```

**Review by wijohnst — COMMENTED** [review]
Left a comment for the README.

File: README.md (hunk 1 above)
Comment (mathiuskitchens) [reply README.md]: Added. Let me know if I missed any steps to get it back up here.

File: README.md [hunk 2]
```diff
@@ -25,3 +25,7 @@ _The Conventional Commit template before adding content, viewed in Vim editor_
 <img width="703" alt="Screen Shot 2023-01-28 at 12 53 16 PM" src="https://user-images.githubusercontent.com/12022922/215283102-a989033d-1ef6-4c88-be1d-24eed49ba618.png">
 
 _The Conventional Commit template after being filled out with all mandatory information_
+
+## Running Locally
+
+``` npm run start ```
```
Comment (wijohnst) [inline README.md]:
```suggestion
```bash
npm run start
```
```

**Review by wijohnst — COMMENTED** [review]
Left a small formatting suggestion.

**Comment by mathiuskitchens** [comment]
> Left a small formatting suggestion.

Did that work?

**Review by wijohnst — APPROVED** [review]
LGTM!

**Pull request merged**